app/views/tradingpremises/summary_details.scala.html

```diff
@@ -21,111 +21,112 @@
+                        case Some(y) => {born @DateHelper.formatDate(y)
+                }
+                case None => {}
+                }
+                </p>
 
-    @model.agentPartnership.map { x =>
-        @checkYourAnswers(
-            question = Messages("tradingpremises.agentpartnership.title"),
-            editLinkTag = "tradingpremisesagentpartnershiptitle-edit",
-            editUrl = controllers.tradingpremises.routes.AgentPartnershipController.get(index, true).toString
-        ) {
-            <p>@x.agentPartnership</p>
+            }
         }
-    }
 
-    @model.agentCompanyDetails.map { x =>
-        @x.companyRegistrationNumber.map { crn =>
+        @model.agentPartnership.map { x =>
             @checkYourAnswers(
-                question = Messages("tradingpremises.youragent.crn"),
-                editLinkTag = "tradingpremisesyouragentcrn-edit",
-                editUrl = controllers.tradingpremises.routes.AgentCompanyDetailsController.get(index, true).toString
+                question = Messages("tradingpremises.agentpartnership.title"),
+                editLinkTag = "tradingpremisesagentpartnershiptitle-edit",
+                editUrl = controllers.tradingpremises.routes.AgentPartnershipController.get(index, true).toString
             ) {
-                <p>@crn</p>
+                <p>@x.agentPartnership</p>
             }
         }
-    }
 
-    @checkYourAnswers(
-        question = Messages("tradingpremises.summary.tradingname"),
-        editLinkTag = "tradingpremisedsummaryaddress-edit",
-        editUrl = controllers.tradingpremises.routes.WhereAreTradingPremisesController.get(index, true).toString
-    ) {
-        <p>@yta.tradingName</p>
-    }
+        @model.agentCompanyDetails.map { x =>
+            @x.companyRegistrationNumber.map { crn =>
+                @checkYourAnswers(
+                    question = Messages("tradingpremises.youragent.crn"),
+                    editLinkTag = "tradingpremisesyouragentcrn-edit",
+                    editUrl = controllers.tradingpremises.routes.AgentCompanyDetailsController.get(index, true).toString
+                ) {
+                    <p>@crn</p>
+                }
+            }
+        }
+
+        @checkYourAnswers(
+            question = Messages("tradingpremises.summary.tradingname"),
+            editLinkTag = "tradingpremisedsummaryaddress-edit",
+            editUrl = controllers.tradingpremises.routes.WhereAreTradingPremisesController.get(index, true).toString
+        ) {
+            <p>@yta.tradingName</p>
+        }
 
         @checkYourAnswers(
             question = Messages("tradingpremises.summary.address"),
             editLinkTag = "tradingpremisedsummaryaddress-edit",
             editUrl = controllers.tradingpremises.routes.WhereAreTradingPremisesController.get(index, true).toString
         ) {
             <p>@yta.tradingPremisesAddress.toLines.map { l =>
-                @l<br />
+                @l <br />
```

Comment:
Is this intentional? Does it render out a space at the end of the line?

Reply:
looks like not needed